QuickshaRe/Domain/QRPictureGenerator.swift

```diff
@@ -6,10 +6,9 @@
 //  Copyright © 2020 Crazism. All rights reserved.
 //
 
-import CoreImage
 import CoreImage.CIFilterBuiltins
 
-final public class QRPictureGenerator {
+public final class QRPictureGenerator {
     
```

Comment:
<!--
  0 failure: 
  1 warning:  Lines should not ...
  
  
  DangerID: danger-id-Danger;
  File: QuickshaRe/Domain/QRPictureGenerator.swift;
  Line: 12;
-->

- :warning: Lines should not have trailing whitespace. (`trailing_whitespace`)